subprojects/build-init/src/main/groovy/org/gradle/api/tasks/wrapper/Wrapper.java

```diff
@@ -63,6 +64,7 @@
     private PathBase distributionBase = PathBase.GRADLE_USER_HOME;
     private String distributionUrl;
     private GradleVersion gradleVersion;
+    private String distType;
```

Comment:
Shouldn't `DistributionType` better go into `DistributionLocator`? IMO the distribution type could be of interest also outside the context of the wrapper, and as such it should not be associated to the wrapper, but to the distribution.